files/jenkins-slave.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+JENKINS_CONFIG=/etc/sysconfig/jenkins-slave
+
+# Read config
+[ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
+
+if [ -x /sbin/runuser ] ; then
+    RUNUSER=runuser
+else
+    RUNUSER=su
```

Comment:
EnvironmentFile doesn't pass the file through a shell interpreter in my understanding, its just for key=value. The logic in the sysconfig file thus wouldn't work.

https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=